src/components/TextInput/TextInput.js

```diff
@@ -248,7 +266,7 @@ class TextInput extends React.Component<Props, State> {
             onChange={this.handleChange}
             onBlur={this.handleBlur}
             onEndEditing={() => this.handleBlur}
```

Comment:
This part looks weird, maybe we should return a function reference like on the line above, or we should actually call the handleBlur function (`onEndEditing={() => this.handleBlur()}`)
Could you check it pls?